packages/@okta/vuepress-site/docs/reference/api/brands/index.md

```diff
@@ -1181,9 +1181,10 @@ Lists all supported email templates
 
 #### Request path parameters
 
-| Parameter      | Type        | Description            |
-| -------------- | ----------- | ---------------------- |
-| `brandId`      | String      | ID of a Brand          |
+| Parameter      | Description            | ParamType | DataType | Required |
+| -------------- | ---------------------- | --------- | -------- | -------- |
+| `brandId`      | ID of a Brand String   | Query     | String   | TRUE     |
+| `expand`       | If specified, it causes additional metadata to be included in the response. Possible values are `settings` and/or `customizationCount`.           | Query     | String    | FALSE     |
```

Comment:
I know the example already illustrates it, but should we mention here that if both are specified, that they should be separated by a comma?

Reply:
done